pep-0215.txt

```diff
@@ -69,7 +69,7 @@ Examples
 ========
 
 Here is an example of an interactive session exhibiting the
-expected behaviour of this feature::
+expected behaviour of this feature. ::
```

Comment:
It might be just me but won't that look odd after the formatting? With a dot appearing right before the colon? I unfortunately can't find any authoritative reference on this but it strikes me as odd because I don't think I've seen it elsewhere. (Similarly for the comma version.)